app/controllers/carts_controller.rb

```diff
@@ -0,0 +1,45 @@
+class CartsController < ApplicationController
+
+  before_action :set_cart, only: [:create, :increment, :decrement]
+
+  def index
+    @cart = Cart.new(session[:cart])
+  end
+
+  def create
+    @cart.add_outing(@outing.id)
+    save_cart(@cart)
+    redirect_to outings_path
+  end
+
+  def increment
+    @cart.add_outing(@outing.id)
+    save_cart(@cart)
+    redirect_to cart_path
```

Comment:
lines 16 & 17 are the same as 10 & 11. You could extract and create a method to DRY up the code.